src/components/App/StepComponentMap.js

```diff
@@ -11,7 +11,7 @@ const stepToComponents = (stepDefaultOptions, {type: stepType, options: stepOpti
       return [<Select method='document' {...optionExt} />,
               <Capture method='document' {...optionExt} autoCapture={true}/>]
     case 'face':
-      return <Capture method='face' {...optionExt} autoCapture={false}/>
+      return <Capture method='face' useWebcam={true} {...optionExt} autoCapture={false}/>
```

Comment:
Yes, although I agree it's unclear. 

This relies on React reading the last argument if an argument is duplicated. Therefore `<Capture method='face' {...optionExt} useWebcam={true} autoCapture={false}/>` would not be customisable.

I considered writing a function to merge useWebcam in to optionExt if it was not already there, but it felt like overkill.